pkg/cmd/scan.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 func NewScanCmd() *cobra.Command {
-	opts := &pkg.ScanOptions{Deep: true}
+	opts := &pkg.ScanOptions{Deep: false}
```

Comment:
Not needed

Reply:
ha yes sorri forgot to rollback after testing :sweat: